### PR TITLE
Admin UI - APIs page

### PR DESCRIPTION
**Pull request by yuval-hazaz**
Show links to GraphQL API, REST API, and Amplication docs on Admin UI sign in page

### CODE REVIEW

**Review by iddan — APPROVED** [review]

File: packages/amplication-data-service-generator/src/admin/app/App.template.tsx [hunk 1]
```diff
@@ -40,6 +43,7 @@ const App = (): React.ReactElement => {
       <MainLayout>
         <Switch>
           <Route path="/login" render={() => <Login onLogin={handleLogin} />} />
+          <Route path="/apis-page" component={ApisPage} />
```
Comment (iddan) [inline packages/amplication-data-service-generator/src/admin/app/App.template.tsx:46]: This route is weird

File: packages/amplication-data-service-generator/src/admin/app/App.template.tsx [hunk 2]
```diff
@@ -74,6 +78,11 @@ const AppLayout = (): React.ReactElement => {
             <CircleBadge name={APP_NAME} />
           </Link>
         }
+        bottomContent={
+          <Link to="/pages/api">
```
Comment (iddan) [inline packages/amplication-data-service-generator/src/admin/app/App.template.tsx:82]: This route is weird

File: packages/amplication-data-service-generator/src/admin/app/App.template.tsx [hunk 3]
```diff
@@ -92,6 +101,8 @@ const AppLayout = (): React.ReactElement => {
         <Page>
           <Switch>
             <PrivateRoute exact path="/" component={Navigation} />
+            <Route path="/pages/api" component={ApisPage} />
```
Comment (iddan) [inline packages/amplication-data-service-generator/src/admin/app/App.template.tsx:104]: This route is weird

**Review by iddan — CHANGES_REQUESTED** [review]
Please move page to /docs/api or something similar

File: packages/amplication-data-service-generator/src/admin/static/src/Login.tsx [hunk 4]
```diff
@@ -39,25 +40,46 @@ const Login = ({ onLogin }: Props) => {
...
+          {error && error.response?.data?.message}
+        </Panel>
+        <div className="divider">Or</div>
+        <Panel panelStyle={EnumPanelStyle.Bordered}>
+          <PanelHeader>Connect via API</PanelHeader>
+          <div className="message">
+            Connect to your app using REST or GraphQL API. Read the docs to
+            learn about authentication and more.
+          </div>
+          <a href="/graphql" target="graphql">
```
Comment (iddan) [inline packages/amplication-data-service-generator/src/admin/static/src/Login.tsx:66]: Should this be Link?

File: packages/amplication-data-service-generator/src/admin/static/src/Login.tsx [hunk 5]
```diff
@@ -39,25 +40,46 @@ const Login = ({ onLogin }: Props) => {
...
+            <Button type="button" buttonStyle={EnumButtonStyle.Secondary}>
+              GraphQL API
+            </Button>
+          </a>
+          <a href="/api" target="api">
```
Comment (iddan) [inline packages/amplication-data-service-generator/src/admin/static/src/Login.tsx:71]: Should this be Link?

File: packages/amplication-data-service-generator/src/admin/static/src/Login.tsx (hunk 4 above)
Comment (yuval-hazaz) [reply packages/amplication-data-service-generator/src/admin/static/src/Login.tsx:66]: yes

File: packages/amplication-data-service-generator/src/admin/static/src/Login.tsx (hunk 5 above)
Comment (yuval-hazaz) [reply packages/amplication-data-service-generator/src/admin/static/src/Login.tsx:71]: yes

**Pull request merged**